src/pages/docs/contributing.md

```diff
@@ -80,22 +145,290 @@ minor tweaks that don’t need extensive testing or review, but for any serious
 coding you really should have a local instance running so you can see your work
 as it progresses.
 
-### Requesting New or Updated Components
+### File structure
 
-If you would like to request changes to a component or a new component be added to Protocol,
-please [submit an issue on Github](https://github.com/mozilla/protocol/issues) using the appropriate
-issue template. Please make sure to fill out this template as thoroughly as possible,
-as this will help the team triage effectively.
+Once you’ve cloned the Protocol repository, you should spend some time
+browsing through the folders in your local copy (or on Github) to
+familiarize yourself with the general structure of the project. You’ll
+have to drill down a bit to reach the really good stuff but the top-level
+root folder looks something like:
 
-### Reporting Bugs
+```
+protocol
+├── bin (scripts that deploy the site to the server)
+├── dist (where source files are built locally for serving [not found in git])
+├── docs (Drizzle documentation)
```

Comment:
This is the only mention of [Drizzle](https://www.trufflesuite.com/docs/drizzle/quickstart) in this document. Should we link/reference it elsewhere?